### PR TITLE
Implement log scale of contour plot

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

Fixes https://github.com/optuna/optuna-dashboard/issues/576

## What does this implement/fix? Explain your changes.

Log axis if distribution is log scale for contour plot

## scripts
```
import logging

import optuna
from optuna_dashboard._app import create_app


host = "127.0.0.1"
port = 8080

optuna.logging.set_verbosity(logging.CRITICAL)


def create_optuna_storage() -> optuna.storages.InMemoryStorage:
    storage = optuna.storages.InMemoryStorage()

    # Single-objective study
    study = optuna.create_study(study_name="single-objective", storage=storage)

    def objective_single(trial: optuna.Trial) -> float:
        x1 = trial.suggest_float("x1", 0, 10)
        x2 = trial.suggest_float("x2", 0.1, 10, log=True)
        x3 = trial.suggest_categorical("x3", ["foo", "bar"])
        return (x1 - 2) ** 2 + (x2 - 5) ** 2

    study.optimize(objective_single, n_trials=100)

    return storage


def main() -> None:
    storage = create_optuna_storage()
    app = create_app(storage, debug=True)
    app.run(host=host, port=port, reloader=True)


if __name__ == "__main__":
    main()
```

### CODE REVIEW

File: optuna_dashboard/ts/components/GraphContour.tsx [hunk 1]
```diff
@@ -278,9 +278,19 @@ const getAxisInfoForNumericalParams = (
...
+    const padding =
+      (Math.log10(distribution.high) - Math.log10(distribution.low)) *
+      PADDING_RATIO
+    min = Math.pow(10, Math.log10(distribution.low) - padding)
+    max = Math.pow(10, Math.log10(distribution.high) + padding)
+  } else {
+    const padding = (distribution.high - distribution.low) * PADDING_RATIO
+    min = distribution.low - padding
+    max = distribution.high + padding
+  }
```
Comment (keisuke-umezawa) [inline optuna_dashboard/ts/components/GraphContour.tsx:293]: ref: to implement those lines, I refered to the optuna implementation. https://github.com/optuna/optuna/blob/master/optuna/visualization/_contour.py#L355-L360

**Comment by c-bata** [comment]
@eukaryo @knshnb Could you review this PR?

**Review by eukaryo — APPROVED** [review]
LGTM.

**Review by knshnb — APPROVED** [review]
Thanks for the PR! I confirmed it works correctly.

**Pull request merged**